app/models/permit_application.rb

```diff
@@ -136,9 +136,13 @@ def permit_type_and_activity
     "#{activity.name} - #{permit_type.name}".strip
   end
 
+  def permit_type_submission_contact
+    jurisdiction.permit_type_submission_contacts.where(permit_type: permit_type)
+  end
+
   def send_submit_notifications
     PermitHubMailer.notify_submitter_application_submitted(submitter, self).deliver_later
-    jurisdiction.users.each { |user| PermitHubMailer.notify_reviewer_application_received(user, self).deliver_later }
+    PermitHubMailer.notify_reviewer_application_received(permit_type_submission_contact, self).deliver_later
```

Comment:
`permit_type_submission_contact` isn't this an array?

Reply:
The jurisdiction has a has_many but I added the above singular method to get the contact for that specific permit based on the jurisdiction AND the permit type

Reply:
Ah yes thank you, will change to a find_by